templates/service-template.yml

```diff
@@ -231,21 +231,11 @@ parameters:
   description: This is the amount of time before a leader lease expires.
   value: "1m"
 
-- name: OBSERVATORIUM_RHSSO_REALM
```

Comment:
My assumption and hope is that this is not used anywhere (in kas-installer and its dependent) I'll approve based on that.

Reply:
See the additional remarks in the original description